style.css

```diff
@@ -105,3 +105,413 @@ body {
+}
+
+.html-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 56px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.boot-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.ruby-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.btn-top {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 126px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal, sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  margin-top: 15px;
+}
+
+.header-pro {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0) 61.94%), url(./images/Img-Placeholder-pro.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 386px;
+  width: 327px;
+  padding: 0 20px;
+}
+
+.pro {
+  width: 295px;
+  padding-top: 84px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.pro-p {
+  width: 255px;
+  padding-top: 18px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.pro-ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  margin-top: 15px;
+}
+
+.pro-html {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 56px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.pro-boot {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.pro-rubby {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.pro-btn {
+  width: 327px;
+  height: 48px;
+  border: none;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.third-sec {
+  width: 375px;
```

Comment:
- [OPTIONAL] Please avoid the use of a limited width size. Don't declare width or use a width of 100% and control the size using margins or paddings. It creates a more responsible page, a better user experience, and works in any kind of display. As not required by the previous reviewers, I consider that optional.
Check how your app could be in this case:

Current app
<img width="822" alt="Screen Shot 2022-08-20 at 9 32 44 AM" src="https://user-images.githubusercontent.com/47673311/185746365-4c890244-6c1d-4077-a3ea-8426dd541c71.png">

Without width limitation:
<img width="822" alt="Screen Shot 2022-08-20 at 9 21 57 AM" src="https://user-images.githubusercontent.com/47673311/185746395-28741731-8af8-4a54-8325-bd7a6fda65d1.png">
<img width="822" alt="Screen Shot 2022-08-20 at 9 22 16 AM" src="https://user-images.githubusercontent.com/47673311/185746398-142315a1-5f35-41b7-ab92-73e08806ed7a.png">